client/ayon_blender/api/plugin.py

```diff
@@ -204,6 +201,9 @@ def cache_instance_data(shared_data):
             cache_legacy = {}
             convert_avalon_instances()
             ayon_instances = bpy.data.collections.get(AYON_INSTANCES)
+            if ayon_instances:
+                ayon_instances.color_tag = "COLOR_04"
+                ayon_instances.use_fake_user = True
```

Comment:
I don't think we should be updating existing collections - the issue with this is that it'll mean it'll always remain forced... if for whatever reason someone changed the color on purpose - this suddenly will revert it?